apps/web/src/components/AccountCard/AccountCard.tsx

```diff
@@ -25,6 +25,7 @@ export const AccountCard = () => {
       <AccountTile
         background={color("50")}
         account={currentAccount}
+        id="account-tile"
```

Comment:
Maybe it's better to put it inside account tile component itself?

Reply:
we need this particular AccoundTile to avoid potential future collisions